### ISSUE TITLE
dependency-locations errors out on components with version specifier

### ISSUE THREAD

**Issue by Ambrevar** [m-n/repl-utilities#8]
Recipe:

```
(repl-utilities:dependency-locations :calispel)
```

Result:

```
Invalid component designator: (VERSION jpl-util 0.2)
```

**Comment by m-n** [comment]
Thank you for the bug report. I think I've addressed this in commit 7e300df, and possibly similar errors complaining about `FEATURE` or `REQUIRE` instead of `VERSION`.

**Comment by Ambrevar** [comment]
It works, thanks a lot!
